functions-and-operators/json-functions/json-functions-aggregate.md

```diff
@@ -0,0 +1,132 @@
+---
+title: JSON Functions That Aggregate JSON Values
+summary: Learn about JSON functions that aggregate JSON values.
+---
+
+# JSON Functions That Aggregate JSON Values
+
+The functions listed on this page are part of the [aggregate functions](/functions-and-operators/aggregate-group-by-functions.md) that TiDB supports, but are specific to working with JSON.
+
+## [JSON_ARRAYAGG()](https://dev.mysql.com/doc/refman/8.0/en/aggregate-functions.html#function_json-arrayagg)
```

Comment:
Do we still need to keep the MySQL docs links in these JSON documents? With detailed descriptions now, how about removing them?

Reply:
I think we should keep that for now. We can remove them at a later stage after we have tested and improved the docs in a few versions.